bin/seeds.js

```diff
@@ -0,0 +1,111 @@
+const mongoose = require('mongoose');
+const Movie = require('../models/Movie.model');
+
+const DB_NAME = "express-cinema";
```

Comment:
Get used to add also the connection string among the variables in the .env file. In a real app this will be the connection string to a remote database